build-tools/packages/build-tools/packlist.txt

```diff
@@ -134,10 +134,6 @@ dist/fluidBuild/tasks/leaf/prettierTask.d.ts
 dist/fluidBuild/tasks/leaf/prettierTask.d.ts.map
 dist/fluidBuild/tasks/leaf/prettierTask.js
 dist/fluidBuild/tasks/leaf/prettierTask.js.map
```

Comment:
I don't get why we have this file. Seems tedious to manage a list of every file we want in the package. I'd assume there is a better way to do this with wild cards or something. Regardless it came up while I was searching for uses of renamer, so I updated it.